utils/completion/daos.bash

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Isn't this still required?

Reply:
Given the permissions on the file, I don't think so.  I think the bash completion stuff sources these files, and doesn't execute them directly.